source/_docs/configuration/yaml.markdown

```diff
@@ -111,3 +111,16 @@ This means that you've mistakenly entered a tab character, instead of spaces.
 Home Assistant is case sensitive, a state of `'on'` is not the same as `'On'` or `'ON'`. Similarly an entity of `group.Doors` is not the same as `group.doors`.
 
 If you're having trouble, check the case that Home Assistant is reporting in the dev-state menu, under *Developer tools*.
+
+### {% linkable_title Can't find configuration.yaml %}
+
+If you're having trouble finding configuration.yaml (or any other configuration files) you can try the following:
+
+1. Login to the server running Hassio (eg via ssh)
```

Comment:
This might be confusing, since if you log in to SSH via the SSH add-on, you'd still not have access to `docker`.